scss/modules/column-navigation.scss

```diff
@@ -124,24 +122,7 @@ $publishedColor: $green;
                     color: $grayLighter;
                 }
 
-                .not-published-state,
-                .not-published {
-                    border: 1px $white solid;
-                    border-radius: 6px;
-                    width: 11px;
-                    height: 11px;
-                    margin-top: 4px;
-                }
-
-                .not-published-state {
-                    background-color: $infoColor;
-                }
-
-                .not-published {
-                    background-color: $publishedColor;
-                }
-
-                .not-published + .not-published-state {
+                .not-published-icon + .not-published-state-icon {
                     margin-left: -11px;
```

Comment:
The column-navigation defines a default space between its icons. Therfore to make the two icons (yellow and green) overlap, the margin-left property needs to be overridden.